website/docs/table-group/data-table/data-table-a11y.md

```diff
@@ -3,20 +3,55 @@ title: DataTable
 tabs: Design('data-table'), Example('data-table-code'), API('data-table-api'), A11y('data-table-a11y'),  Changelog('data-table-changelog')
 ---
 
+## What component has
+
+### Keyboard support
+
+Table: Keyboard support
+
+| Key                                                   | Function                                              |
+| ----------------------------------------------------- | ----------------------------------------------------- |
+| `Tab`                                                 | Moves focus to the next focusable element.            |
+| `Shift + Tab`                                         | Moves focus to the previous focusable element.        |
+| `Left Arrow`, `Right Arrow`, `Up Arrow`, `Down Arrow` | Move the focus across the grid of table cells.        |
+| `Enter`                                               | Activates interactive elements inside the table grid. |
```

Comment:
```suggestion
| `Enter`, `Space`                                               | Activates interactive elements inside the table grid. |
```
I guess Space as well, if there are any buttons, inputs, etc

Reply:
Added, but I will put it in one of my last commits. This suggestion doesn't want to apply